examples/demo-stf/src/sov-cli/main.rs

```diff
@@ -0,0 +1,200 @@
+use anyhow::Context;
```

Comment:
We need a `README.md` for this tool. We had one for `bank-cmd` but this one here will have to be a bit more generic.